contrib/harfbuzz/module.defs

```diff
@@ -18,11 +14,7 @@ HARFBUZZ.CONFIGURE.extra = \
     FREETYPE_CFLAGS="-I$(call fn.ABSOLUTE,$(CONTRIB.build/))include/freetype2" \
 
 ifneq ($(BUILD.system),darwin)
```

Comment:
Will do. Should I perhaps also merge the `--with-fontconfig=no` line into the previous section?